routes/api.js

```diff
@@ -1,44 +1,34 @@
 'use strict';
 
-var apiConfig = require('../server/config').api;
-var COCKTAIL_PATH = apiConfig.base + apiConfig.cocktailPath;
-var dbOperations = require('../server/db-operations');
+const apiConfig = require('../server/config').api;
+const COCKTAIL_PATH = `${apiConfig.base}${apiConfig.cocktailPath}`;
+const dbOperations = require('../server/db-operations');
 
-module.exports = function (app) {
+module.exports = (app) => {
```

Comment:
A "fat arrow" function! This is equivalent to `module.exports = function (app) {...`.

Key differences:
* Doesn't create a new `this`
* Doesn't create a new `arguments` object
* Cleaner syntax in some cases

Because (in my experience at least) we pretty much never want a new `this`, arrow functions are the new default.

P.S. In this specific case, we don't even need the parens around `app`: they're optional when there's just one var, but mandatory when there are 0, 2, etc. (That is, `module.exports = app => {...` would work.) But it's more obvious what's happening when _all_ functions take parens, so that's how the eslint is set up.